test/1_test_fullpath.js

```diff
@@ -84,14 +87,881 @@ contract('Voucher tests', async (addresses) => {
+            ev._to === constants.ZERO_ADDRESS &&
+            ev._id.eq(tokenSupplyKey2) &&
+            ev._value.eq(new BN(1))
+          );
+        },
+        'transfer single event incorrect'
+      );
+
+      truffleAssert.eventEmitted(
+        internalTokenTx,
+        'Transfer',
+        (ev) => {
+          return (
+            ev._from === constants.ZERO_ADDRESS &&
+            ev._to === users.buyer.address &&
+            ev._tokenId.eq(tokenVoucherKey)
+          );
+        },
+        'transfer event incorrect'
+      );
+
+      //Check BosonRouter state
+      assert.equal(
+        await contractBosonRouter.correlationIds(users.buyer.address),
+        1,
+        'Correlation Id incorrect'
+      );
+
+      //Check Voucher Kernel state
+      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+        tokenVoucherKey
+      );
+      assert.isTrue(voucherStatus.status.eq(new BN(128))); //128 = COMMITTED
+      assert.isFalse(
+        voucherStatus.isPaymentReleased,
+        'Payment released not false'
+      );
+      assert.isFalse(
+        voucherStatus.isDepositsReleased,
+        'Deposit released not false'
+      );
+
+      //Check ERC1155ERC721 state
+      const sellerERC1155ERC721Balance = await contractERC1155ERC721.balanceOf(
+        users.seller.address,
+        tokenSupplyKey2
+      );
+      assert.isTrue(sellerERC1155ERC721Balance.eq(new BN(0)));
+
+      const buyerERC721Balance = await contractERC1155ERC721.balanceOf(
+        users.buyer.address
+      );
+      const erc721TokenOwner = await contractERC1155ERC721.ownerOf(
+        tokenVoucherKey
+      );
+      assert.isTrue(buyerERC721Balance.eq(new BN(1)));
+      assert.strictEqual(users.buyer.address, erc721TokenOwner);
+    });
+
+    it('must fail: adding new order with incorrect value sent', async () => {
+      await truffleAssert.reverts(
+        contractBosonRouter.requestCreateOrderETHETH(
+          [
+            constants.PROMISE_VALID_FROM,
+            constants.PROMISE_VALID_TO,
+            constants.PROMISE_PRICE1,
+            constants.PROMISE_DEPOSITSE1,
+            constants.PROMISE_DEPOSITBU1,
+            constants.ORDER_QUANTITY1,
+          ],
+          {
+            from: users.seller.address,
+            to: contractBosonRouter.address,
+            value: 0,
+          }
+        ),
+        truffleAssert.ErrorType.REVERT
+      );
+    });
+
+    it('must fail: fill an order with incorrect value', async () => {
+      await truffleAssert.reverts(
+        contractBosonRouter.requestVoucherETHETH(
+          tokenSupplyKey1,
+          users.seller.address,
+          {
+            from: users.buyer.address,
+            to: contractBosonRouter.address,
+            value: 0,
+          }
+        ),
+        truffleAssert.ErrorType.REVERT
+      );
+    });
+  });
+
+  describe('Vouchers (ERC721)', function () {
+    beforeEach('setup contracts for tests', async () => {
```

Comment:
We might need to change the comment in the `beforeEach`  where possible

Reply:
Changed to "execute prerequisite steps". Will be included in next commit.